enhancements/20240604-lvm-csi-driver.md

```diff
@@ -0,0 +1,77 @@
+# LVM CSI Driver
+
+## Summary
+
+In some specific scenarios, we need better performance and could use a strict location for the volume. The LVM could be a good choice because it is a thin layer on top of the block devices. Also, it provides useful features like volume expansion and snapshots. We want to introduce the LVM CSI driver to support the LVM volume in Harvester. 
+
+### Related Issues
+
+https://github.com/harvester/harvester/issues/5724
+
+## Motivation
+
+We do not have any current solution to support local volumes, which means directly using the local disk as the volume. The LVM CSI driver could help leverage the local disk as the volume. If the user could use the strict location VM, it would help us improve performance and latency.
+
+### Goals
+
+Introduce the LVM CSI driver to support the LVM volume in Harvester.
+
+### Non-goals [optional]
```

Comment:
Remove the word "[optional]" here, as we are explicitly stating some non-goals :-)

Reply:
Cool, thanks!  Updated!